components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -376,9 +372,8 @@ void AdsServiceImpl::StartBatAdsService() {
   CHECK(!IsBatAdsServiceBound());
 
   bat_ads_service_remote_ = bat_ads_service_factory_->Launch();
-
   bat_ads_service_remote_.set_disconnect_handler(base::BindOnce(
-      &AdsServiceImpl::RestartBatAdsServiceAfterDelay, AsWeakPtr()));
+      &AdsServiceImpl::Shutdown, AsWeakPtr()));
```

Comment:
Maybe we should add a DisconnectHabdler function and log that this occurred before calling shutdown to help diagnose any issues.